README.md

```diff
@@ -172,7 +170,7 @@ The possible sections are:
 - Main section: `[main]`
   - `path_filter_fat`: whether to filter path components for FAT file systems
   - `path_filter_special`: whether to filter path components for special characters
-  
+
```

Comment:
This is just gonna create merge conflicts with https://github.com/JoeLametta/whipper/pull/240 -- mind putting the whitespace back?

Reply:
Should be fixed.